Source/WTF/wtf/CompactVariant.h

```diff
@@ -110,6 +118,11 @@ template<CompactVariantCapable V> class CompactVariant {
         Operations::destruct(m_data);
     }
 
+    void swap(CompactVariant& other)
+    {
+        std::swap(m_data, other.m_data);
+    }
```

Comment:
Do we also want an overload of `std::swap` that does this or a free function `WTF::swap`? Trying to remember the best practice.